rw-system.sh

```diff
@@ -198,11 +198,27 @@ if [ "$(getprop ro.vendor.product.manufacturer)" = "motorola" ] || [ "$(getprop
     fi
 fi
 
-if getprop ro.vendor.build.fingerprint | grep -q -i -e xiaomi/wayne -e xiaomi/jasmine; then
+if getprop ro.vendor.build.fingerprint | grep -q -i -e xiaomi/wayne -e xiaomi/jasmine -e xiaomi/jasmine_sprout; then
```

Comment:
jasmine matches jasmin_prout, so is it really useful?

Reply:
Ig not but it was defined earlier. Either remove both checks for _sprout or make them conform.